app/controllers/assessments_controller.rb

```diff
@@ -181,34 +181,46 @@ def update
 
     assessment = Assessment.find_by(id: params.permit(:id)[:id])
     reported_symptoms_array = params.permit({ symptoms: %i[name value type label notes required] }).to_h['symptoms']
+    redirect_to(root_url) && return if reported_symptoms_array.map { |symptom| symptom[:name] }.include?(:nil)
```

Comment:
Can you just use `reported_symptoms_array.any?` rather than `.map`, `.include?`

Reply:
this is specifically checking for `nil` symptom names instead of the array being empty, so that would be different

Reply:
ah I knew something looked a little off about that, it should be `nil`

Reply:
done: https://github.com/SaraAlert/SaraAlert/pull/852/commits/62cf0ea0e118c6ade7cc94db30b101b4ae23f4c4